### PR TITLE
ChunkViewModel: Delete unused materials

### PR DESCRIPTION
**Pull request by manavortex**
# ChunkViewModel: Delete unused materials

Tried and tested, works!

### CODE REVIEW

File: WolvenKit.App/ViewModels/Shell/ChunkViewModel.cs [hunk 1]
```diff
@@ -58,7 +58,7 @@ public partial class ChunkViewModel : ObservableObject, ISelectableTreeViewItemM
 {
     private readonly IChunkViewmodelFactory _chunkViewmodelFactory;
     private readonly IDocumentTabViewmodelFactory _tabViewmodelFactory;
-    private readonly ILoggerService _loggerService;
+    public readonly ILoggerService _loggerService;
```
Comment (seberoth) [inline WolvenKit.App/ViewModels/Shell/ChunkViewModel.cs:61]: Please don't. Get a new instance of LoggerService where needed

**Review by seberoth — CHANGES_REQUESTED** [review]

File: WolvenKit.App/ViewModels/Shell/ChunkViewModel.cs (hunk 1 above)
Comment (manavortex) [reply WolvenKit.App/ViewModels/Shell/ChunkViewModel.cs:61]: I need it in the RedTreeView - can you help me out?

Comment (seberoth) [reply WolvenKit.App/ViewModels/Shell/ChunkViewModel.cs:61]: Use `Locator.Current.GetService` in Views. Just look up one of the other usages to see how

Comment (manavortex) [reply WolvenKit.App/ViewModels/Shell/ChunkViewModel.cs]: OK, thanks, learned a thing :) Fixed

**Review by seberoth — APPROVED** [review]

**Pull request merged**